lib/server/ServerRenderer.js

```diff
@@ -68,26 +68,28 @@ function makeClientAppStartScript(clientAppRequirePath, environmentConfig, boots
         "clientApp.start({\n" +
         "environmentConfig: " + environmentConfig + ",\n" +
         "bootstrappedData: " + bootstrappedData + "\n" +
-    "});";
+        "});";
 }
 
 function makeBootstrappingLoader(scriptBody, clientAppUrl, isAsync) {
     var loadAsync = isAsync ? "s.async = true;\n" : "";
 
     return "<script type=\"text/javascript\">\n" +
-        "\"use strict\";" +
-        "(function(d) {\n" +
-        "var s = d.createElement(\"script\");\n" +
+        "var s = document.createElement(\"script\");\n" +
+        "var h = document.head;\n" +
         "s.setAttribute(\"src\", \"" + clientAppUrl + "\");\n" +
         loadAsync +
         "function loaded(e) {\n" +
-            scriptBody + "\n" +
-            "s.removeEventListener(\"load\", loaded, false);\n" +
+        scriptBody + "\n" +
+        "s.removeEventListener(\"load\", loaded, false);\n" +
+        "s = null;\n" +
         "}\n" +
         "s.addEventListener(\"load\", loaded);\n" +
-        "setTimeout(function() { (d.body || d.head).appendChild(s); }, 0);\n" +
-    "})(document);\n" +
-    "</script>";
+        "setTimeout(function(){\n" +
+        "h.insertBefore(s, null);\n" +
+        "h = null;\n" +
+        "});\n" +
```

Comment:
No browsers throw an error if you omit it, but I guess to be technically correct I should add it.